src/extensions/replay/sessionrecording.ts

```diff
@@ -21,6 +21,8 @@ import { _timestamp, loadScript } from '../../utils'
 
 import { _isBoolean, _isNull, _isNumber, _isObject, _isString, _isUndefined } from '../../utils/type-utils'
 import { logger } from '../../utils/logger'
+import { getRecordNetworkPlugin } from './network/record'
+import { buildNetworkRequestOptions } from './network/record/default-options'
```

Comment:
Should we move all of this stuff (aside from maybe some types) into the recorder loader?